Source/HTTP/Android/http_android.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
do we need a header for this? i guess we probably do if we want to reference in the other file, yeah?

Reply:
yeah, can't reference the JNI calls in the other file otherwise